pygeoapi/api.py

```diff
@@ -2317,6 +2334,34 @@ def get_collection_coverage(self, request: Union[APIRequest, Any],
 
             headers['Content-Type'] = collection_def['format']['mimetype']
             return headers, 200, data
+        elif format_ == F_HTML:
+            LOGGER.debug(data)
+
+            data['id'] = dataset
+            data['title'] = l10n.translate(
+                self.config['resources'][dataset]['title'],
+                self.default_locale)
+            data['collections_path'] = self.get_collections_url()
+
+            # Convert the CRS URL into a Proj4 string, since the client
+            # needs the later and we have the Proj DB
+            data['proj4str'] = ""
+            if data['crs'] == "http://www.opengis.net/def/crs/OGC/1.3/CRS84":
+                data['proj4str'] = "+proj=longlat +datum=WGS84 +no_defs"
+            else:
+                match = re.compile("http:\/\/www.opengis.net\/def\/crs\/(?:(?:OGC)|(?:EPSG))\/.*\/\/?(\d+)").match(data['crs'])
+
+                if match != None:
```

Comment:
We need to move this functionality to the rasterio provider, with the idea that projection handling should be done in the provider.